pkg/operator/ceph/object/admin.go

```diff
@@ -162,12 +164,19 @@ func extractJSON(output string) (string, error) {
 // RunAdminCommandNoMultisite is for running radosgw-admin commands in scenarios where an object-store has not been created yet or for commands on the realm or zonegroup (ex: radosgw-admin zonegroup get)
 // This function times out after a fixed interval if no response is received.
 func RunAdminCommandNoMultisite(c *Context, expectJSON bool, args ...string) (string, error) {
```

Comment:
It might be worth commenting that it returns `kerrors.IsNotFound(err)` if the remote pod isn't found,